koalas-requirements.txt

```diff
@@ -1,4 +1,4 @@
 pyspark>=3.0.0,<3.2.0
-koalas>=1.8.1,<=1.8.2
+koalas>=1.8.0,<=1.8.2; python_version < '3.10'
```

Comment:
Not necessary since FT doesn't support 3.10 yet.

Reply:
updated